src/store/middleware/model-poller.ts

```diff
@@ -177,14 +177,22 @@ export const modelPollerMiddleware: Middleware<
         reduxStore.dispatch
       );
       return response;
-    } else if (action.type === jujuActions.findAuditEvents.type) {
-      // Intercept findAuditEvents actions and fetch and return audit events via the
+    } else if (action.type === jujuActions.fetchAuditEvents.type) {
```

Comment:
Instead of returning the result of the API request to the component that dispatched the action, this now fetches the audit events and then dispatches `updateAuditEvents(...)` to store the result in Redux.